test-data/unit/check-bound.test

```diff
@@ -46,7 +46,7 @@ z = G(B())
 
 
 [case testBoundVoid]
-# flags: --no-strict-optional
+# flags: --no-strict-optional --no-local-partial-types
```

Comment:
Making `--local-partial-types` invertible will certainly help which it's adoption. E.g. we could also consider adding it to `--strict` as a first step before making it the default in `2.0`.

I do just wonder what change made the test diffs necessary. Adding `add_invertible_flag` with `default=False` shouldn't change anything, does it?

_It will probably make sense to enable it for all our tests by default, but in that case the test config should probably be changed in the same PR. E.g. it could be added here._
https://github.com/python/mypy/blob/55d4c1725bae29ad5ac2ce857b4b4b3363e5518c/mypy/test/testcheck.py#L128-L131

--
Feel free to split that up into a different PR if you like though.

Reply:
Oh, it wasn't necessary, I just proactively added it to test cases where I felt confident it was what we needed

Reply:
Yeah, changing default in tests is not a bad idea